site/guide/_manage-permissions.qmd

```diff
@@ -0,0 +1,3 @@
+1. Under User Permissions, select **Permissions**
+
+1. Under your desired action, add or delete roles. 
```

Comment:
How do I this fam, I've never seen the interface before, where do I click, give me some more context, please give me a step-by-step, or maybe even just break it down into bullet points:

e.g.
```
Manage roles that have authorization to perform actions:

- To assign a role to a permission, blah blah blah
- To remove a role from a permission, blah blah blah```

Reply:
To add to what Beck said, the steps should cover that you click **+** to add a role to the permission or remove a role by clicking on the **x** on the label: 

<img width="991" alt="image" src="https://github.com/validmind/documentation/assets/15148011/b42c1883-8afa-498e-b86e-7add2252b60b">